### ISSUE TITLE
Add shuffle all for favorite tracks

### ISSUE THREAD

**Issue by Rednas83** [nukeop/nuclear#1182]
Preferably by artist/genre/all

Something like for example groove. Which also allows sorting and filtering
![image](https://user-images.githubusercontent.com/25219229/152131981-78754198-f69a-44b2-87df-dc75153dc443.png)

**Comment by nukeop** [comment]
Do you mean just adding them all to the queue with shuffle enabled?

**Comment by Rednas83** [comment]
Yes, that would be awesome! A 1 click way for playing shuffled music depending on a genre filter. Play all by double clicking an album/Artist would also make a lot of Sense I think.

**Comment by aaron-liao-604** [comment]
hey @nukeop it's my first time contributing and I've read through the contributing guidelines. I'm wondering if this is something I can pick up?

**Comment by nukeop** [comment]
Sure, I'll assign you. Just let me know how you're planning to approach this.

**Comment by nukeop** [comment]
This issue is from an older version of Nuclear that has been replaced with a complete rewrite. None of the old issues apply to the new codebase.

The old code is preserved on the [legacy/electron](https://github.com/nukeop/nuclear/tree/legacy/electron) branch. If you have feedback about the new version, head to [Discussions](https://github.com/nukeop/nuclear/discussions).
